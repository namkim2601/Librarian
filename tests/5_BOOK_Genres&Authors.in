LiSt GeNrEs
LiSt AuThOrs
GeNrE No Such Genre
AuThOr No Such Author
add collection collection.csv
list genres
list authors
genre No Such Genre
genre poetry
genre Historical Fiction
author No Such Author
author Douglas Adams
exit
