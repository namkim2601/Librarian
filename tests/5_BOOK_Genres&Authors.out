user: No books in system.

user: No books in system.

user: No books in system.

user: No books in system.

user: 17 books successfully added.

user: Comedy
Comic
Epic
Fantasy
Historical Fiction
Horror
Poetry
War

user: Bob Kane
Douglas Adams
Harper Lee
Homer
J.K. Rowling
J.R.R. Tolkien
Jamie Lee Curtis
T.S. Eliot
Virgil
W.E. Johns

user: No books with genre No Such Genre.

user: The Iliad (Homer)
The Odyssey (Homer)
Wasteland (T.S. Eliot)

user: To Kill a Mockingbird (Harper Lee)

user: No books by No Such Author.

user: The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)

user: Ending Library process.
